.github/workflows/release.yml

```diff
@@ -17,7 +17,20 @@ jobs:
           # Requires "Read and Write access to code" permission
           token: ${{ secrets.RELEASE_ACTION_ACCESS_TOKEN }}
 
+      - name: Check for changes since last tag
+        id: check_changes
+        run: |
+          LATEST_TAG=$(git tag --sort=-authordate --merged=main | head --lines 1)
```

Comment:
I believe this needs to sort by commit date in case we merge a commit that was authored before some other commit but only merged afterwards.
Probably an edge case that might never arise but this seems safer regardless.

Reply:
Tested on pull_request. Switched to `committerdate` sorting.